Add crand explain (#3140)
open-telemetry/opentelemetry.io: content/en/docs/collector/trace-receiver.md

}
```

> #### Check your work
>

> > - Importing crypto/rand package as crand because name is duplicated with
> >   math/rand.

Reviewer: ```suggestion
> - Imported the `crypto/rand` package as `crand` to avoid name conflict with 
>   math/rand.
```

Author: Applied suggestion. Thank you

> 

Reviewer 2: It is my understanding that "check your work" sections highlight additions or changes that were mentioned previously.

In this case, there is no sample code that illustrates the import.

I'd suggest that we instead add the required import to the code block that ends at line 1919, and then drop this "Check your work" call out.

Reviewer: yes, let's have that partial import

Reviewer: @chalin please take another look
